scripts/mkstatic.py

```diff
@@ -1263,6 +1318,145 @@ def write(self, metadata_request, metadata_define, arguments, debug):
                             if container == var.container:
                                 break
 
+
+                        (dimensions_target_name, dim_string_target_name) = extract_dimensions_from_local_name(var.target)
+
+                        # Derive correct horizontal loop extent for this variable for the rest of this function
+                        if var.rank:
+                            array_size = []
+                            dim_substrings = []
+                            for dim in var.dimensions:
+                                # DH* TODO
```

Comment:
This needs to be addressed in a follow-up PR, because we can't switch the ufs-weather-model and other models using potentially inactive variables to contiguous arrays without fixing the current way we handle them.